src/renderer/components/Preference/Index.vue

```diff
@@ -57,6 +57,8 @@
     bottom: 0;
     left: auto;
     z-index: 10;
+    width: -webkit-fill-available;
+    margin-right: 20.1rem;
```

Comment:
Magical code...the `margin-right` value is 78+200+8 ?

Reply:
`margin-right` value isn't necessary but I added it so that the scroll bar is visible from the bottom. I used the Developer Tools to determine the value and it seemed to persist no matter how I resized the window.

Reply:
Sorry, I missed the padding of form-actions

78+200+8+36 = 322
322 / 16 = 20.125 rem

Reply:
Is there another way to achieve this? I am quite inexperienced when it comes to `css` styling.

Reply:
I tried using vue slot, but it was a bit of a hassle and I didn't finish it.

Reply:
Using slots would require quite a few changes to the preferences code. I am fairly new to `vue.js`, if you point me in the right direction, maybe I can look into implementing it.